content/en/docs/concepts/configuration/secret.md

```diff
@@ -1041,10 +1051,10 @@ for secret data, so that the secrets are not stored in the clear into {{< glossa
  - Applications still need to protect the value of secret after reading it from the volume,
    such as not accidentally logging it or transmitting it to an untrusted party.
  - A user who can create a pod that uses a secret can also see the value of that secret.  Even
-   if apiserver policy does not allow that user to read the secret object, the user could
+   if `apiserver` policy does not allow that user to read the secret object, the user could
    run a pod which exposes the secret.
  - Currently, anyone with root on any node can read _any_ secret from the apiserver,
```

Comment:
I would write this as:
```suggestion
  - Anyone with superuser access to any node can read **any** Secret from the API server.
```
and drop the mention of the planned feature - the style guide recommends that you [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

Optionally, put that revised sentence in a `{{< caution >}}` block.